samples/snippets/sample_default_values.py

```diff
@@ -115,7 +115,7 @@ def disable_usage_export(project_id: str) -> None:
     # Updating the setting with None will disable the
```

Comment:
Please update the comment to reflect the change to the sample.

Reply:
Good catch! Fixed.